.hyperagent/census-2026-06-27/census_synthesis.py

```
@@ -0,0 +1,205 @@
+alpha_full = cm.krippendorff_alpha(R, distance=cm.nominal_distance)
+
+# core+shoulder only: drop the 6 door-local tail nodes (named by <=2 readers)
+keep = [j for j,nm in enumerate(node_names) if len(NODES[nm]) >= 3]
+R_core = [[R[c][j] for j in keep] for c in range(N)]
+alpha_core = cm.krippendorff_alpha(R_core, distance=cm.nominal_distance)
+
+# bootstrap CI for full alpha by resampling UNITS (nodes) with replacement
+def boot_alpha(mat, ncols, B=3000):
+    vals=[]
+    for _ in range(B):
+        cols=[random.randrange(ncols) for _ in range(ncols)]
```

Ruff (0.15.18)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)